lib/token.ts

```diff
@@ -272,6 +273,16 @@ function handleOAuthResponse(sdk: OktaAuth, tokenParams: TokenParams, res: OAuth
       };
     }
 
+    if (refreshToken) {
+      tokenDict.refreshToken = {
+        refreshToken: refreshToken,
+        value: refreshToken,
```

Comment:
let's avoid "value". We have an existing task to remove this. https://github.com/okta/okta-auth-js/pull/515

Reply:
I started that and saw that it'd be changing the base type of Token so I held off, but I can give it another go.

Reply:
#515 is a breaking change since type would be changed. btw, can anyone approve that pr.

Reply:
updating this comment for posterity: Decision was to leave .value to keep this from being a breaking change, but we'll remove it from everything in 5.0